### PR TITLE
fix(android): Add minor improvements to android getting started guide

### PR DESCRIPTION
**Pull request by sdhuka**
### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [X] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] iOS
- [X] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [X] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://link.com)` 
            HTML: `<a href="https://link.com">link</a>`_

### When this PR is ready to merge, please check the box below
- [X] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

**Review by abdallahshaban557 — APPROVED** [review]
LGTM!

File: src/fragments/lib/project-setup/android/create-application/40_verifyAmplifyLibraries.mdx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Create an `Application` class and add the Amplify initialization into its `onCreate()` to initialize Amplify once in your application.
+Create a class by extending `Application` and override its `onCreate()` to to initialize Amplify in your application.
```
Comment (gpanshu) [inline src/fragments/lib/project-setup/android/create-application/40_verifyAmplifyLibraries.mdx:1]:
```suggestion
Create a class by extending `Application` and override its `onCreate()` to initialize Amplify in your application.
```

**Review by gpanshu — CHANGES_REQUESTED** [review]
minor comment

**Review by dengdan154 — APPROVED** [review]

**Review by abdallahshaban557 — APPROVED** [review]
LGTM

**Review by gpanshu — APPROVED** [review]

**Review by dengdan154 — APPROVED** [review]

**Review by hdworld11 — APPROVED** [review]

**Review by katiegoines — APPROVED** [review]

**Pull request merged**